start-database.sh

```diff
@@ -8,7 +8,11 @@ echo "Adding data into MySQL"
 file -bi /var/mysql/database.sql
 /usr/sbin/mysqld &
 sleep 5
-curl $url | mysql --default-character-set=utf8
+curl $url -o import.sql
+sed -ri.bak 's/-- Database: (.*?)/CREATE DATABASE \1/g' import.sql
```

Comment:
I think the CREATE DATABASE statement has to close with a ; to generate valid SQL code. SQL statements have to be separated by a ';'